### ISSUE TITLE
Correlate a picture to the generated temperature

### ISSUE THREAD

**Issue by dozoyo** [bottleware/weather#7]
Depending on what the api returns, we can share pictures of a cloudy day, a rainy day, whatever seems to be the coolest picture background. This will give a nice touch.

It doesn't have to be a full background either, maybe it can just be really simple, like icons.

A cloudy icon, rainy icon. Change in the background color depending on it. 

Or maybe like some clouds show up at the top if it's cloudy/rainy/whatever, or a sun.

In short, lotsa options I'll put into one sentence each:
A. Real pictures of the weather described in the area
B. Icons or single color props that convey the current weather
C. Background color changes

I think for the most interesting outcome, a combination of B & C.
